### PR TITLE
Init README & sample .eslintrc

### PR DESCRIPTION
**Pull request by mikeliu32**
# What? Why?
Init README & sample .eslintrc
# How was it tested?
localhost
# Depend on

cc
@pennyyi @hden @bardichezanber @JasmineEZ @Kellyvei @nosecoolie

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,115 @@
 # ezstandard
 EZTABLE JavaScript standard
+
+## Install
```
Comment (hden) [inline README.md]: I don't think this section is necessary as most of dev here do not need to setup projects. Perhaps we should place this in the wiki?

**Comment by hden** [comment]
An for those who use [vim](https://github.com/editorconfig/editorconfig-vim), [emacs](https://github.com/editorconfig/editorconfig-emacs), [sublime](https://github.com/sindresorhus/editorconfig-sublime), or [atom](https://atom.io/packages/editorconfig), please save the world by sharing your [editorconfig](http://editorconfig.org) at the wiki.

**Comment by hden** [comment]
@mikeliu32 Overall, awesome job. :+1:

All we need to do is push a little bit harder to make it a ~~0-config~~ **it-just-works** :sparkles: experience.

**Comment by mikeliu32** [comment]
@hden thanks for the feedback! I'll update it later :+1:

**Comment by JasmineEZ** [comment]
begging config for sublime!

**Comment by hden** [comment]
@JasmineEZ see https://github.com/EZTABLE/ezstandard/pull/4

**Comment by JasmineEZ** [comment]
Nice, thanks @hden

**Comment by hden** [comment]
If no further objection has been raised before 12/21. This will be merged and made available for the company.

**Comment by mikeliu32** [comment]
I'll update this PR before the end of this week!  這兩天一直沒時間修 😅

**Pull request merged**